tests/ui/attributes/no_link/multiple-crates-and-no_link.rs

```diff
@@ -0,0 +1,17 @@
+//! Regression test for #13560. Previously, it was possible to
+//! trigger an assert in crate numbering if a series of crates
+//! being loaded included a "syntax-only" extern crate.
+//! But it appears we don't mess with crate numbering for
+//! `#[no_link]` crates anymore, so this test doesn't seem
+//! to test anything now.
```

Comment:
Remark: I think we'll leave this as an exercise to the person modifying `#[no_link]`-the-attribute...

Reply:
Let's not rub it in ;0